From f3646608832288bb08b705de9b54a04a618a1ce3 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 5 Mar 2024 12:41:30 +0100
Subject: [PATCH] mkosi: make shellcheck happy

(cherry picked from commit e86b1a9b0f1d2fc150edde18ea6f748b9423df3c)

Related: RHEL-27512
---
 mkosi.images/base/mkosi.build.chroot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkosi.images/base/mkosi.build.chroot b/mkosi.images/base/mkosi.build.chroot
index c46d667a90..69430c304d 100755
--- a/mkosi.images/base/mkosi.build.chroot
+++ b/mkosi.images/base/mkosi.build.chroot
@@ -8,7 +8,7 @@ set -e
 # We don't want to install our build of systemd in the base image, but use it as an extra tree for the
 # initrd and system images, so override DESTDIR to store it in the output directory so we can reference it as
 # an extra tree in the initrd and system image builds.
-DESTDIR="$OUTPUTDIR/systemd"
+export DESTDIR="$OUTPUTDIR/systemd"
 
 # If mkosi.builddir/ exists mkosi will set $BUILDDIR to it, let's then use it
 # as out-of-tree build dir. Otherwise, let's make up our own builddir.
